### PR TITLE
Add bio_Verousca

### PR DESCRIPTION
**Pull request by Verousca**
# Student Bio PR

Write a short bio for yourself using Markdown and add it to your class
repository using Local/Remote workflows. Just a little intro so your classmates
can get to know you.

### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [x] any embedded code snippets are formatted

### CODE REVIEW

**Comment by bermarte** [comment]
A tip: don't always write 'fix a bug' for your commits, try to be more specific.

File: student-bios/Verousca_bio.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Hobbies
+
+- Travelling
+- Listening to Music
+- Escape room
+  
+## Photo
+
+![This is a alt text.](images/Verousca_Img.jpeg "This is a sample image.")
```
Comment (bermarte) [inline student-bios/Verousca_bio.md]: It should be` /student-bios/images/Verousca_Img.jpeg`

**Review by bermarte — COMMENTED** [review]
Hi, there is a minor change for the image to be done

File: student-bios/Verousca_bio.md (hunk 1 above)
Comment (Verousca) [reply student-bios/Verousca_bio.md]: Thanks for the feedback. I have made the corrections.

**Review by bermarte — APPROVED** [review]

**Comment by bermarte** [comment]
The file's name should be `Verousca.md`

**Pull request merged**